### DISCUSSION TITLE
Bug when simply sending a form

### DISCUSSION THREAD

**Discussion by batata004** [fomantic/Fomantic-UI#1487] [Behaviors (Api, Form Validation, Visibility)] [answered]
The simplest PoC of this bug is here -> http://quemfazsite.com.br/temp/teste.html

Click inside the field "your name" and press ENTER on your keyboard. According to the documentation at "https://semantic-ui.com/behaviors/form.html" when pressing enter the form should submit and "Submit events will be attached to click on any element inside the form with class submit".

In the form on the link I posted above, there are 2 buttons: the red one is a normal button and the blue one is the one that HAS the `submit` class. Despite that, when I press enter on the form (inside the "your name" field) it executes the RED button onclick event and not the BLUE SUBMIT button.

However, if I change the order of those 2 buttons, it triggers the BLUE one when trying to send the form using ENTER. So clearly there is a but cause when the `.submit` button is not the first one in the form, it is not getting triggered when user presses ENTER inside any field of the form.

**Comment by ko2in** [comment, marked as answer]
@batata004 the red button is also a submit button by default if you do not declare via `type="button"` or `type="reset"`.

**Comment by batata004** [comment]
You are damm right! I just double checked the documentation and every button, by default, is always a submit! Thanks!
